src/gc/env/gcenv.ee.h

```diff
@@ -71,6 +71,9 @@ class GCToEEInterface
 
     static void HandleFatalError(unsigned int exitCode);
     static bool ShouldFinalizeObjectForUnload(AppDomain* pDomain, Object* obj);
+    static MethodTable* GetFreeObjectMethodTable();
+    static size_t GetSizeOfArrayBase();
```

Comment:
These methods are called once on GC startup and cached on the EE side - is that still a problem?